package.json

```diff
@@ -11,7 +11,8 @@
     "install:packages": "pnpm install --filter='!./apps/*'",
     "package": "turbo run package --filter='./packages/*'",
     "package:graph": "turbo run package --filter='./packages/*' --graph",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "format": "prettier --write \"**/*.{ts,svelte,md}\"",
+    "prettier": "prettier --check \"**/*.{ts,svelte,md}\"",
```

Comment:
hmm it's hit all the svelte files but what about the astro and mdx files?

Reply:
I considered leaving that for a follow up since this already resulted in a lot of lint changes 😆

Reply:
fair enough,  must admit when I first opened the PR I was like.... "87 files changes for CI/CD" 😱 ...and then realized it was just the newly formatted files

Reply:
I'm also fine with a follow up for that. Formatting in mdx files is pretty tricky and sometimes prettier is just messing up the output, so it might not be desirable after all.